src/codegen/llvm/codegen_amdgpu.cc

```diff
@@ -84,7 +88,11 @@ class CodeGenAMDGPU : public CodeGenLLVM {
         llvm::GlobalVariable *global = new llvm::GlobalVariable(
             *module_, type, false, llvm::GlobalValue::PrivateLinkage, 0, ".shared",
             nullptr, llvm::GlobalValue::NotThreadLocal, shared_address_space);
+#if TVM_LLVM_VERSION >= 100
+        global->setAlignment(llvm::Align(info.alignment));
```

Comment:
Should we use `MaybeAlign`?

https://llvm.org/doxygen/classllvm_1_1AllocaInst.html#a98fa120553a55845f6cac450c30788d1

Reply:
`MaybeAlign` is basically `optional<Align>` https://github.com/llvm/llvm-project/blob/master/llvm/include/llvm/Support/Alignment.h#L13-L17
Should be no difference in our case?